doc/content/getting-started/migrating-from-v2/major-changes.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Major Changes in The Things Stack
+weight: 10
+---
+
+Before getting started, we will discuss major breaking changes between {{% ttnv2 %}} and {{% tts %}}, along with some guidelines to make the migration process easier to manage.
+
+<!--more-->
+
+## Architecture
+
+Behind the scenes, {{% tts %}} is using a new architecture. See {{< ref "/reference/components" >}} for a complete overview of the different components of {{% tts %}}.
```

Comment:
See `[Components]({{< ref "/reference/components" >}})` for a complete overview

Reply:
Fixed